src/renderers/shaders/ShaderLib/sprite_vert.glsl

```diff
@@ -11,19 +11,24 @@ void main() {
 
 	#include <uv_vertex>
 
+	vec4 mvPosition = modelViewMatrix * vec4( 0.0, 0.0, 0.0, 1.0 );
+
 	vec2 scale;
 	scale.x = length( vec3( modelMatrix[ 0 ].x, modelMatrix[ 0 ].y, modelMatrix[ 0 ].z ) );
 	scale.y = length( vec3( modelMatrix[ 1 ].x, modelMatrix[ 1 ].y, modelMatrix[ 1 ].z ) );
 
+	#ifndef USE_SIZEATTENUATION
+
+		scale *= abs( mvPosition.z );
```

Comment:
We can use ```scale *= projectionMatrix[2][3] * mvPosition.z``` so that the orthographic camera will have correct result. ```projectionMatrix[2][3] * mvPosition.z``` is the gl_position.w value.

Reply:
@06wj  `projectionMatrix[ 2 ][ 3 ]` is zero for an orthographic camera, so that will set the scale to zero. 

However, this is a good approach. I'll add it in a separate PR and we can discuss it. Thank you for the suggestion.

Reply:
My mistake. 

The scale by camera is ```(projectionMatrix * mvPosition).w```, equals ```vec4(projectionMatrix[0][3], projectionMatrix[1][3], projectionMatrix[2][3], projectionMatrix[3][3]) * mvPosition```.

So we have two solution:
```
if(projectionMatrix[ 2 ][ 3 ] == -1.0){
    scale *= abs( mvPosition.z );
}
```
or
```
scale *= abs( vec4(projectionMatrix[0][3], projectionMatrix[1][3], projectionMatrix[2][3], projectionMatrix[3][3]) * mvPosition );
```